api/src/pcapi/routes/backoffice/collective_offers/collective_offer_templates_blueprint.py

```diff
@@ -416,9 +416,13 @@ def get_collective_offer_template_details(collective_offer_template_id: int) ->
     collective_offer_template_query = educational_models.CollectiveOfferTemplate.query.filter(
         educational_models.CollectiveOfferTemplate.id == collective_offer_template_id
     ).options(
-        sa.orm.joinedload(educational_models.CollectiveOfferTemplate.venue).joinedload(
-            offerers_models.Venue.managingOfferer
-        )
+        sa.orm.joinedload(educational_models.CollectiveOfferTemplate.venue)
+        .joinedload(offerers_models.Venue.confidenceRule)
+        .load_only(offerers_models.OffererConfidenceRule.confidenceLevel),
+        sa.orm.joinedload(educational_models.CollectiveOfferTemplate.venue)
+        .joinedload(offerers_models.Venue.managingOfferer)
+        .joinedload(offerers_models.Offerer.confidenceRule)
+        .load_only(offerers_models.OffererConfidenceRule.confidenceLevel),
```

Comment:
Détail (optionnel) : pour éviter la répétition de `venue`, je crois que tu peux écrire : 
```suggestion
        sa.orm.joinedload(educational_models.CollectiveOfferTemplate.venue).options(
            sa.orm.joinedload(offerers_models.Venue.confidenceRule)
            .load_only(offerers_models.OffererConfidenceRule.confidenceLevel),
            sa.orm.joinedload(offerers_models.Venue.managingOfferer)
            .joinedload(offerers_models.Offerer.confidenceRule)
            .load_only(offerers_models.OffererConfidenceRule.confidenceLevel)
        )
```
Je ne l'ai découvert que tard, d'où les innombrables répétitions dans le code existant, mais ça permet de mieux visualiser l'enchaînement des relations chargées.

Reply:
Bon à savoir !
On pourrait factoriser à plusieurs endroits en effet